Makefile.am

```diff
@@ -43,12 +43,20 @@ $(distdir)/ChangeLog: check-git-tool
 	  LANG=C git log -- $(top_srcdir) > $(distdir)/ChangeLog ; \
 	fi
 
+warn-pre-built-man:
```

Comment:
I think this general warning is to much. As we build the manpages from `man/*.in` and not from `*.md`

Reply:
No, we build from .md when pandoc is available. Only when pandoc is not available, we omit the .md -> .in step
and then warn when the dist archive is prepared for release with "make package" or "make rpmpackage" (with distcheck and other checks).

Reply:
Ok I had different expectation. But I guess it makes sense to (re-)build the manpages with pandoc, if we do not explicit disable it with `./configure --disable-pandoc`